### ISSUE TITLE
Invites not being sent

### ISSUE THREAD

**Issue by IronWrath** [oculus-samples/Unreal-SharedSpaces#20]
Hi, my invites never seen to get received on my other headset.

I walk up to roster / invite panel. Invite my test account with the SharedSpaces app, App says the invitation has been sent but its never received on the other side. I have also checked Settings > Notifications > Social > Friends Activity is turned ON and do not disturb is OFF on both sides. If I make both of the headsets go into the purple room I can see the other one, briefly.  The Query Me and Query Friends panels seem to work, just not the roster and invite panels. I sometimes also get a "You are no longer in this lobby message".

**Comment by els-oculus** [comment]
The first thing that I would verify is that you have completed the data use checkup for your app:
https://developer.oculus.com/resources/publish-data-use/

Here's a screenshot of what it looks like for invites:

![duc](https://user-images.githubusercontent.com/61444568/172471334-266f1f66-191b-44ac-a276-bfcee939e58c.png)

**Comment by yvasquezmonoku** [comment]
If you are uploading it in alpha, the person you invite must be the developer of the project. At least in our case this worked.

**Comment by IronWrath** [comment]
> If you are uploading it in alpha, the person you invite must be the developer of the project. At least in our case this worked.

I initially had them as a admin, I swapped them other to developer and the issue is still the same.

**Comment by IronWrath** [comment]
> The first thing that I would verify is that you have completed the data use checkup for your app: https://developer.oculus.com/resources/publish-data-use/
> 
> Here's a screenshot of the Oculus dashboard for one of our apps, to show you what it looks like for invites:
> 
> ![duc](https://user-images.githubusercontent.com/61444568/172471334-266f1f66-191b-44ac-a276-bfcee939e58c.png)

I have the necessary permissions enabled and approved (all of them at this point). The issue still remains the same.

**Comment by els-oculus** [comment]
Have you resolved this issue?

**Comment by ry3nwa** [comment]
> Have you resolved this issue?

The issue regarding the invites seemed to be a networking issues (university WiFi), works via a hotspot.  The "You are no longer in this lobby message", is still an issue and could be because of the firmware as mentioned above, one of them is a couple of years old (the one the message appears on), the other is quite new (1 - 3 months). I've just resorted to using public rooms which is sufficient for the purposes I need it for.

**Comment by els-oculus** [comment]
Yes, old firmware will definitely cause issues.   Please close if this is resolved.
